### ISSUE TITLE
function deepCopy cannot copy dates

### ISSUE THREAD

**Issue by neomastertlt** [effector/effector#975]
https://github.com/effector/effector/blob/e10fb7d9d1e4e6171b4eb24ffa4740a6477bfad2/src/vue/lib/deepCopy.ts#L6

if you call a function and pass an object containing a date as an argument, the function will not work correctly
console.log(deepCopy({ a: new Date() }));
{ a: new Date() } => { a: {} }

**Comment by zerobias** [comment]
Yes, in this case library supports only primitives, as it's hard to support each case, we will switch to native structuredClone when it will have broader support

**Comment by falkomerr** [comment]
![image](https://github.com/effector/effector/assets/105551615/1f929faa-4f7c-4f8a-b6de-2ba594982b05)
Now, structuredClone has 93% support
